### DISCUSSION TITLE
REALITY 使用 dl.google.com，IP 被封锁报告

### DISCUSSION THREAD

**Discussion by christarcher** [XTLS/Xray-core#2302] [Q&A]
{
            "listen": "0.0.0.0",
            "port": 443,
            "protocol": "vless",
            "settings": {
                "clients": [
                    {
                        "id": "....",
                        "flow": "xtls-rprx-vision"
                    }
                ],
                "decryption": "none"
            },
            "streamSettings": {
                "network": "tcp",
                "security": "reality",
                "realitySettings": {
                    "show": false,
                    "dest": "dl.google.com:443",
                    "xver": 0,
                    "serverNames": ["dl.google.com"],
                    "privateKey": "....",
                    "minClientVer": "1.8.0",
                    "shortIds": ["6c6872"]
                }
            },
            "sniffing": {
                "enabled": true,
                "destOverride": [
                    "http",
                    "tls"
                ]
            }
        }
回国流量走的是warp 服务器除了reality没有其他协议
今天晚上使用reality用了没多久之后就被封了，是整个ip都被封禁不是封端口，ping.pe国内都不通
我感觉可能是和使用的dest有关
是不是因为dl.google.com在国内有服务器，然后由于dns的问题解析成了国内的服务器 ？
dns填的1.1.1.1 服务器在hk

**Comment by RPRX** [comment]
**指向 dl.google.com 就是会被封，群里讨论过，原因是境内有服务器，正常你应该访问它，而不应出境，且 GFW 本就追着境外 Google IP 封**

这么说吧，**dl.google.com 的 IP 可能还放着 www.google.com 等**，~~你还转发它，GFW 不封你封谁，留着过年？这都不封的话，翻墙多简单~~

相关链接：域名推荐 https://github.com/XTLS/Xray-core/discussions/2256#discussioncomment-6295296 ；uTLS Chrome 指纹的 dl.google.com 问题 https://github.com/XTLS/Xray-core/issues/2219#issuecomment-1595774823

---

目前我们在中国收到的 REALITY 被封报告都经不起追问，**要么是自己配置不当，要么是甲骨文之类的**，所以我在一些地方偶尔看到有人说 REALITY 也会被封，觉得挺那啥的，**总是有些小白自己不会用，封了又怪 REALITY，逢人就来一句我 REALITY 也封了，就是不说配置**

还偶尔看到有人仅拿 REALITY 和其它协议比延迟、速度，**我想说 REALITY 就个没代理功能的安全层，比啥呀，这些都取决于你的具体配置**，比如你 REALITY 上用的什么协议，**有没有多路复用，有没有 XTLS，`dest` 是什么、离多远 https://github.com/XTLS/Xray-core/issues/2017#issuecomment-1532345891** ，表现都是不一样的

**Comment by RPRX** [comment]
~~GitHub 这 Discussions 太简陋了，改个标题都没记录~~

原标题：使用reality ip被封锁报告

**Comment by ccckfg** [comment]
填国外网站的域名时最好检查一下这些域名有没有国内CDN,我之前看到有些人的 REALITY `dest`指向`www.microsoft.com`,这个域名在国内也有CDN

**Reply by malagebidi** [reply]
CDN有香港的，安全吗？

**Reply by toyo2333** [reply]
> CDN有香港的，安全吗？

同问

**Reply by ccckfg** [reply]
香港应该没问题,不过最好你的节点服务器也在香港

**Reply by toyo2333** [reply]
> 香港应该没问题,不过最好你的节点服务器也在香港

这确实是理想状态，但很多时候选的域名它们自己也是cdn，是解析到cf到IP或者亚马逊的IP。比如一个日本某景点官网域名，dns解析得到的是 香港、新加坡、美国的亚马逊IP。

**Reply by toyo2333** [reply]
其实，这个问题也是因为传播过程导致的，一些一键脚本，xui面板等默认就是Microsoft。这就没办法了。

**Comment by evanfury** [comment]
谷歌算哪门子的白名单 不是重点关照对象吗

**Comment by toyo2333** [comment]
> 谷歌算哪门子的白名单 不是重点关照对象吗

网上一些文章都教程、一些一键脚本的默认配置里，就是谷歌和微软的域名。。这怪谁呢 哈哈
